src/component/mxgraph/shape/render/IconPainter.ts

```diff
@@ -19,6 +19,11 @@ import BpmnCanvas from './BpmnCanvas';
 import StyleUtils from '../../StyleUtils';
 import { IconStyleConfiguration, ShapeConfiguration, Size } from './render-types';
 
+export enum IconConstants {
```

Comment:
:warning: I would
- move the values to the existing `StyleDefault` enum
- rename them: remove the plural on the `icon` part for consistency with the rest of the naming + something like `SHAPE_ACTIVITY_MARKER_MARGIN`

Reply:
agreed